### DISCUSSION TITLE
post/authentik-tutorial-5-setup-migration/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#16] [Blog Post Discussion]
# post/authentik-tutorial-5-setup-migration/

Homepage and blog for Zhenghao Wu (ECWU), sharing thoughts on technology, everyday experiences, and photography. Engaging with the community through knowledge sharing.

https://ecwuuuuu.com/post/authentik-tutorial-5-setup-migration/

**Comment by WowYiJiu** [comment]
postgres用postgres-backup-local镜像备份挺好的  
[github地址](https://github.com/prodrigestivill/docker-postgres-backup-local)
[docker镜像地址](https://hub.docker.com/r/prodrigestivill/postgres-backup-local)
